src/user/formats/hal.ts

```diff
@@ -40,6 +39,13 @@ export function item(user: User, privileges: PrivilegeMap) {
     type: TypeMap.get(user.type),
     privileges
   };
+
+  if (user.type === 3) {
+    hal._links.members = {
```

Comment:
Usually when we link to a single resource that is a collection, the style we use would be something like 'member-collection' instead of 'members'.

I'm not sure how consistent we've been with this, but it makes spelling certain links less awkward, especially when the singular word already ends with and s.